src/Symfony/Component/Routing/Matcher/UrlMatcher.php

```diff
@@ -201,6 +201,7 @@ protected function getAttributes(Route $route, string $name, array $attributes):
             unset($defaults['_canonical_route']);
         }
         $attributes['_route'] = $name;
+        $attributes['_path'] = $route->getPath();
```

Comment:
The name should be `_route_path` to avoid confusion with `Request::getPath()` and the `_path` query parameter used for fragments.

Reply:
done